src/main/java/net/mcreator/ui/modgui/LootTableGUI.java

```diff
@@ -49,7 +49,7 @@ public class LootTableGUI extends ModElementGUI<LootTable> {
 
 	private final JComboBox<String> type = new JComboBox<>(
 			new String[] { "Block", "Entity", "Generic", "Chest", "Fishing", "Empty", "Advancement reward", "Gift",

```

Comment:
Check https://misode.github.io/loot-table/, it appears Dispensers is not really a valid type. Dispener type is actually chest type so it can't really be added like this.